httpbp/server.go

```diff
@@ -45,10 +45,14 @@ type httpHandlerFactory struct {
 }
 
 func (f httpHandlerFactory) NewHandler(endpoint Endpoint) http.Handler {
-	wrappers := make([]Middleware, 0, len(f.middlewares)+len(endpoint.Middlewares)+1)
+	wrappers := make([]Middleware, 0, len(f.middlewares)+len(endpoint.Middlewares)+2)
```

Comment:
probably should add a comment to explain the magic number of 2 😅 (it's for `SupportedMethods` + `recoverPanic`, if I'm reading it correctly)

Reply:
personal preference, for magic numbers I think its best to use a `const` and comment the const definition.